recipe/meta.yaml

```diff
@@ -1,17 +1,17 @@
-{% set version = "6.4" %}
+{% set version = "6_4_20240210" %}
```

Comment:
This was intentional as `6.4` is not the same as `6.4.20240210`. Otherwise we might have updated patch level without a clear indication what version it is.